packages/dashboard/src/run/runSummary/runSummary.tsx

```diff
@@ -23,6 +23,17 @@ import { RunStartTime } from '../runStartTime';
 import { RunSummaryTestResults } from '../runSummaryTestResults';
 import { RunTimeoutChip } from '../runTimeoutChip';
 
+function getTotalFromSpecResults(specs?: any[]) {
+  if (!specs) return undefined;
+  let total = 0;
+
+  specs.forEach((spec) => {
+    total += spec.results?.stats?.tests ?? 0;
+  });
+
+  return total;
+}
+
```

Comment:
I can't see this function is referenced anywhere. Do we need it?

Reply:
@ImanMahmoudinasab You are correct, it's a remainder of when I was trying to figure out why the total eventually reported results differed from what cypress initially tells it's going to execute.